### DISCUSSION TITLE
Aufgabe 3: nur bestimmte Attribute extrahieren

### DISCUSSION THREAD

**Discussion by kaiserog** [irgroup-classrooms/dis08-2021#70] [Q&A] [answered]
Guten Tag,

Müssen wir nur bestimmte Attribute extrahieren, z.B. 5 Stück, oder ist es auch okay, die gesamte Infobox zu extrahieren und anschließend alle ungewünschten Spalten/Attribute zu löschen?

Die Frage gilt im Allgemeinen auch für das Cleaning:

Muss das data cleaning im webscraping Prozess geschehen, oder ist es auch okay das nachträglich zu machen z.B. mit Pandas?

Beste Grüße,
Konstantin Kaiser

**Comment by phschaer** [comment, marked as answer]
Lieber Herr Kaiser, das steht Ihnen absolut frei. Wie auch immer Sie sich entscheiden, es ist gut zu begründen, warum Sie sich für eine der Varianten entschieden haben und was für Vor- und Nachteile das mit sich bringt.
